libs/design-system/src/lib/Pages/LearnPage/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { ReactComponent as Filter } from '../../../../images/action/filter2.svg';
+import { LandingHeader } from '../../Components/LandingHeader';
+import { SocialLinksSignup } from '../../Components';
+import { ButtonLink } from '../../Components/Button';
+import { TutorialCardProps } from '../../Components/TutorialCard';
+import PageSections from '../shared/PageSections';
+import PageSection from '../shared/PageSection';
+import { useCallback, useMemo, useState } from 'react';
+import {
+  LargeVideoCard,
+  LargeVideoCardProps,
+} from '../../Components/VideoCard/LargeVideoCard';
+import {
+  SmallVideoCard,
+  SmallVideoCardProps,
+} from '../../Components/VideoCard/SmallVideoCard';
+import { ToggleButton } from '../../Components/ToggleButton';
+import { PaginatedTutorialCardList } from '../../Components/TutorialCard/PaginatedTutorialCardList';
+
+export type LearnPageProps = {
+  allTutorials: TutorialCardProps[];
+  cadenceHref: string;
+  cadenceTutorials: TutorialCardProps[];
+  nftTutorials: TutorialCardProps[];
+  videos: {
+    primary: LargeVideoCardProps;
+    secondary: SmallVideoCardProps[];
+  };
+};
+
+export function LearnPage({
+  allTutorials = [],
+  cadenceHref,
+  cadenceTutorials,
+  nftTutorials,
+  videos,
+}: LearnPageProps) {
+  const [filters, setFilters] = useState<string[]>([]);
+
+  const allTutorialsFiltered = useMemo(
+    () =>
+      filters.length > 0
+        ? allTutorials.filter(({ tags }) =>
+            filters.some((filter) => tags.includes(filter))
+          )
+        : allTutorials,
+    [allTutorials, filters]
+  );
+
+  const tags = Array.from(
+    new Set(allTutorials.reduce((acc, { tags }) => [...acc, ...tags], []))
+  ).sort();
+
+  return (
+    <div className="container w-full bg-primary-gray-50 p-6 dark:bg-black">
```

Comment:
The container class will have to be moved to the individual page section components so that the gray background can be full width. I also added a <PageBackground> component that can replace this div